spec/lib/droplet_kit/paginated_resource_spec.rb

```diff
@@ -73,4 +73,18 @@
       end
     end
   end
-end
\ No newline at end of file
+
+  describe '#==' do
```

Comment:
Are you missing a spec here? Comparisons with empty and equal are fine, but what about not equal?